lib/LoginStrategy.ts

```diff
@@ -424,6 +486,28 @@ export class LoginStrategy extends BaseStrategy {
     return true;
   }
 
+  isSiweMessagePayload(payload: unknown): payload is ISiweMessagePayload {
+    if (!payload) return false;
+    if (typeof payload !== 'object') return false;
+    const payloadKeys = Object.keys(payload);
+    if (
+      !hasIn(payload, 'domain') &&
```

Comment:
All these fields are required
```suggestion
      !hasIn(payload, 'domain') ||
      !hasIn(payload, 'nonce') ||
      !hasIn(payload, 'uri') ||
      !hasIn(payload, 'address') ||
      !hasIn(payload, 'chainId') ||
      !hasIn(payload, 'version')
```